### ISSUE TITLE
dhcp6 server stalls if ddns script fails

### ISSUE THREAD

**Issue by aerusso** [RMerl/asuswrt-merlin.ng#489]
EDIT: The issue was caused by a ddns update script (that made an ssh call) failing. The script did not have the correct host key (because of the addition of ed25519 key support).

This is a "wishlist" bug, asking for a noisier failure mode, and more robust treatment of a hung ddns updater script

1. Please provide some visual indication that part of the dhcp is malfunctioning.
2. If the ddns updater script fails, the dhcp6 server shouldn't fail.

I'm leaving the below, for other people experiencing similar symptoms.

[I'm troubleshooting losing ipv6 connectivity after my ipv6 address expires](https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/411#note_464063).  This probably started after I upgraded to 384.16_0 on my RT-AC68U (RT-1900P).  There are logs there from my client.

In summary: my dhcp6 client (on the computer) claims that it has sent `INFORMATION-REQUEST`, (ipv6) but eventually times out, never receiving a response.  This only began interfering with my ipv6 connectivity after flashing the upgrade and rebooting the router.

However: I have logs that suggest this may have been going on much longer than before 384.16_0, but I cannot confirm the behavior without flashing back to an earlier version. 

Is it safe to flash back to 384.15_0? I can troubleshoot this if someone can let me know if it's safe to flash back.

~I think this may be related to #488.~ In retrospect, this is completely unrelated.

**Comment by gxing** [comment]
I flash back to 384.15. 
My device is RT-AC66U_B1.  
Upgrade from 384.15 to 384.16.
I do not know if your device can safely flash back.
